### DISCUSSION TITLE
OPPO RENO5 can‘t add database

### DISCUSSION THREAD

**Discussion by xuxumu** [pawelsalawa/sqlitestudio#4119] [Q&A]
### Details
Here you describe your problem details, or proposal for enhancement.

### Steps to reproduce
I am using two mobile phones, one is Huawei Mate20pro, the other is OPPO RENO5 Pro, the database of adding OPPO RENO5 Pro can‘t be found, Mate20pro can be found, please tell me how to solve it, thank you！
![微信图片_20210521093417](https://user-images.githubusercontent.com/65522352/119068950-c100c780-ba17-11eb-88c2-3b9bf3ae8d0f.png)

### Operating system
windows10

### SQLiteStudio version
v3.3.3

**Comment by pawelsalawa** [comment]
It's hard to tell why.
Is the database "bounded" to the application, or is it placed on SD card? (SD card databases are not yet supported by this plugin)
Is your application built as "Debug" version, or "Release" version? (the Release version cannot be debugged)
